src/types.js

```diff
@@ -62,6 +62,13 @@ export type Identifier = PatternBase & {
   __clone(): Identifier;
 };
 
+export type PrivateName = PatternBase & {
+  type: "PrivateName";
+  name: string;
+
+  __clone(): Identifier;
```

Comment:
I think it's only `this.#a`, `#a` and `#a = 1` in the class body? At least from the examples.